api/test/unit/index.test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
Can this be placed in a directory that matches the source tree so we know which index.js you are testing?
E.g., /test/unit/models/index.test.js